### DISCUSSION TITLE
Mac App Store (MAS)

### DISCUSSION THREAD

**Discussion by resoltico** [cyanfish/naps2#468] [General]
_Hello,_

If NAPS2 is already in the store indeed, as the release log suggests, can anyone post the link, please — I cannot find the listing by searching for "NAPS2".

_Thank you_

**Comment by cyanfish** [comment]
It's in review at the moment, it might be a few days.

**Reply by Floesse** [reply]
I was trying to search for in App Store, but did not find it, can you let us know when it will be available?

Also under which name it will be in the App Store?

Thanks in advance.

**Comment by resoltico** [comment]
Apparently it is taking much longer than expected, as it is still not on MAS.
